docs/release-notes/2.24.0.md

```diff
@@ -0,0 +1,65 @@
+# Percona Monitoring and Management 2.24.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">November 18, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+## Release Highlights
+- **Better PostgreSQL monitoring**
+  - Starting from this version of PMM, it will support monitoring of PostgreSQL 14. Both versions - Community and [Percona Distribution for PostgreSQL](https://www.percona.com/software/postgresql-distribution).
+  - This version of PMM has also a significant improvement in exposed data and additional features if you monitor PostgreSQL together with the [pg_stat_monitor](https://github.com/percona/pg_stat_monitor) extension (Part of Percona distribution for PostgreSQL). We added such features:
+    - The PostgreSQL queries will have complete Query execution plan information. This will help with future optimization of queries and get a clear understanding of query performance 
+    - Query execution histograms collection and presentation inside Query Analytics for a much more detailed understanding of query executions. 
+    - Query analytics will show and let the user drill down to Top Query for the particular query if it's subquery and have this parent query. This feature will allow the user to see the dependencies between queries better and understand the impact of subqueries. 
+    - Query Analytics can filter PostgreSQL queries by query commands like SELECT, UPDATE, etc., and by Application Name if it's set for PostgreSQL connection from the application.
+
+
+- **Integrated Alerting (Technical preview):**
+    - Alerting in PMM now has an additional notification channel type - webhooks. So now, users can integrate Alerting with any tool they use for Incident management.  Read more about new notification channels and how to set up them in our documentation (LINK to DOC) 
```

Comment:
can we link to the current netlify page?

Reply:
this one doesn't have those docs. But sure temporary you can link to the:   https://deploy-preview-626--pmm-doc.netlify.app/using/alerting.html#add-a-notification-channel

Reply:
```suggestion
    - Alerting in PMM now has an additional notification channel type - webhooks. So now, users can integrate Alerting with any tool they use for Incident management.  Read more about new notification channels and how to set up them in [our documentation  ](https://deploy-preview-626--pmm-doc.netlify.app/using/alerting.html#add-a-notification-channel)
```

Reply:
needs to be updated for public release